Program.cs

```diff
@@ -15,7 +16,7 @@ static void Main(string[] args)
 
     class Fairytale

```

Comment:
Please, try to follow the rule: one type - one file.

Reply:
Moved